docs/wikidata.md

```diff
@@ -27,12 +29,14 @@ build the index under a different path.
 ## Download and uncompress Wikidata
 
 If you already downloaded **and decrompressed** Wikidata to uncompressed Turtle
-format you can skip this step, otherwise we download and uncompress it.
+format you can skip this step. Otherwise we download and uncompress it.
```

Comment:
`to uncompressed` -> `to the uncompressed`
I'm probably just nitpicking at this point, but the `Otherwise we download and uncompress it.` still sounds slightly weird to me. What about `Otherwise we'll download and uncompress it in this step.`